File: OnlinerMoto/VehicleItemDetails.m
//
//  VehicleItemDetails.m
//  OnlinerMoto
//
//  Created by Igor Karpov on 2.11.2013.
//  Copyright (c) 2013 KarpovIV. All rights reserved.
//

#import "VehicleItemDetails.h"

@implementation VehicleItemDetails

- (id)initWithVehicleItem:(VehicleItem *)vehicleItem
{
    self = [super init];
    
    if (self)
    {
        self.vehicleItem = vehicleItem;
    }
    
    return self;
}

@end

File: OnlinerMoto/VehicleItemDetails.swift
//
//  VehicleItemDetails.swift
//  OnlinerMoto
//
//  Created by Igor Karpov on 12.11.2014.
//  Copyright (c) 2014 KarpovIV. All rights reserved.
//

import Foundation

class VehicleItemDetails : NSObject
{
    var vehicleItem: VehicleItem?
    var additionalDescription: String?
    var location: String?
    var allPhotos: Array<NSData>?

    override init() {}
    
    init(vehicleItem: VehicleItem)
    {
        self.vehicleItem = vehicleItem
    }
}
